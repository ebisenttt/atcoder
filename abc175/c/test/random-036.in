744055254327942 147328398352654 69541928620127
