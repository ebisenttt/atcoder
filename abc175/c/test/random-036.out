48635968126672
